src/main/java/org/humancellatlas/ingest/export/UpdateExporter.java

```diff
@@ -0,0 +1,88 @@
+package org.humancellatlas.ingest.export;
+
+import org.humancellatlas.ingest.biomaterial.BiomaterialRepository;
+import org.humancellatlas.ingest.bundle.BundleManifest;
+import org.humancellatlas.ingest.bundle.BundleManifestService;
+import org.humancellatlas.ingest.core.MetadataDocument;
+import org.humancellatlas.ingest.core.MetadataDocumentMessageBuilder;
+import org.humancellatlas.ingest.core.Uuid;
+import org.humancellatlas.ingest.core.web.LinkGenerator;
+import org.humancellatlas.ingest.file.FileRepository;
+import org.humancellatlas.ingest.messaging.MessageRouter;
+import org.humancellatlas.ingest.process.ProcessRepository;
+import org.humancellatlas.ingest.project.ProjectRepository;
+import org.humancellatlas.ingest.protocol.ProtocolRepository;
+import org.humancellatlas.ingest.submission.SubmissionEnvelope;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class UpdateExporter implements Exporter {
+
+    @Autowired
+    private BundleManifestService bundleManifestService;
+
+    @Autowired
+    private ProjectRepository projectRepository;
```

Comment:
Should be able to use the MetadataCrudService here rather than importing all the Repository beans